hack/jenkins/e2e.sh

```diff
@@ -161,7 +160,6 @@ AWS_REQUIRED_SKIP_TESTS=(
 
 # Tests which kills or restarts components and/or nodes.
 DISRUPTIVE_TESTS=(
-    "Autoscaling\sSuite.*scale\scluster"
```

Comment:
I'm not sure about this change.
Scaling cluster is a disruptive test and I think it should be left here.